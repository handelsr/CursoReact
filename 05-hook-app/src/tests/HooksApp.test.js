import { HookApp } from '../HooksApp';
import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

describe('Pruebas en <HookApp/>', () => {
    test('debe de ser igual al snapshot', () => {
        const wrapper = shallow(<HookApp/>)

        expect(wrapper).toMatchSnapshot();
    })
})
