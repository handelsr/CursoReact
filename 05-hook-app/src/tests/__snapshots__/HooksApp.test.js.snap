// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp/> debe de ser igual al snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
